porcupine/plugins/highlight/tree-sitter-token-mappings/python.yml

```diff
@@ -30,7 +30,7 @@ queries:
   class_definition: |
     (class_definition
       ("class") @Token.Keyword
-      name: (identifier) @Token.Name.Function
+      name: (identifier) @Token.Name.Function  # close enough lol
```

Comment:
Actually, idk why I made this `Token.Name.Function` probably some copy-pasta, it could be `Token.Name.Class`.